src/languageservice/services/schemaRequestHandler.ts

```diff
@@ -45,20 +45,6 @@ export const schemaRequestHandler = (connection: IConnection, uri: string): Then
     });
   }
 
-  // vscode schema content requests are forwarded to the client through LSP
-  // This is a non-standard LSP extension introduced by the JSON language server
-  // See https://github.com/microsoft/vscode/blob/master/extensions/json-language-features/server/README.md
-  if (scheme === 'vscode') {
-    return connection.sendRequest(VSCodeContentRequest.type, uri).then(
-      (responseText) => {
-        return responseText;
-      },
-      (error) => {
-        return error.message;
-      }
-    );
-  }
-
```

Comment:
Can you elaborate more about deletting this code?

Reply:
Yeah, I found out it's not actually used anywhere. I think it must have came in when I originally had copied the schemaRequestHandler over from the json language server. I looked into what it's used for on VSCode and it's just used for setting their schemas for the settings, keybindings, etc. Since YAML ls doesn't need any of those specific things it doesn't make sense to keep